Mathlib/Order/Interval/Set/Basic.lean

```diff
@@ -1181,6 +1181,10 @@ theorem Ioo_union_Icc_eq_Ioc (h₁ : a < b) (h₂ : b ≤ c) : Ioo a b ∪ Icc b
 
 /-! #### Two finite intervals, `I?c` and `Io?` -/
```

Comment:
This comment is outdated now, can you fix it, please?

Reply:
I decided to move the new lemma to the following section.